### PR TITLE
Added details about quotas

### PR DESCRIPTION
**Pull request by seandavi**

### CODE REVIEW

**Comment by seandavi** [comment]
I signed it!

File: README.md [hunk 1]
```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
     (https://cloud.google.com/genomics/v1alpha2/pipelines).
 *   Read about [Dataflow](https://cloud.google.com/dataflow).
 *   Write your own workflows!
+
+## A word about quotas
+
+Google has numerous quotas that may impact the parallel execution of large numbers of tasks.  Some hints include:
```
Comment (jbingham) [inline README.md]: Google Cloud Platform has various quotas that affect how many VMs and IP addresses, and how much disk space you can get. Some tips:

File: README.md [hunk 2]
```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
...
+
+* Check and potentially increase quotas here: https://console.cloud.google.com/compute/quotas
+* Consider using as many zones as possible (eg., 'us-*')
```
Comment (jbingham) [inline README.md]: Consider listing all zones in the region or geography where your Cloud Storage bucket is (e.g., for standard buckets in the EU, use "eu-*"; for regional buckets in US central, use "us-central-*").

File: README.md [hunk 3]
```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
     (https://cloud.google.com/genomics/v1alpha2/pipelines).
 *   Read about [Dataflow](https://cloud.google.com/dataflow).
 *   Write your own workflows!
+
+## A word about quotas
+
+Google has numerous quotas that may impact the parallel execution of large numbers of tasks.  Some hints include:
+
+* Check and potentially increase quotas here: https://console.cloud.google.com/compute/quotas
```
Comment (jbingham) [inline README.md]: [Check and potentially increase quotas](https://console.cloud.google.com/compute/quotas)

**Comment by seandavi** [comment]
I made changes as suggested in my repo.  I have not done this type of code review before.  Do I simply do a new pull request at this point?  Or is there a way to update the pull request to include the changes?  Sorry for the naive question.

**Comment by jbingham** [comment]
You did it right! You updated the pull request.


On Mon, Sep 26, 2016 at 8:00 AM Sean Davis <notifications@github.com> wrote:

I made changes as suggested in my repo. I have not done this type of code
review before. Do I simply do a new pull request at this point? Or is there
a way to update the pull request to include the changes? Sorry for the
naive question.

—
You are receiving this because you commented.


Reply to this email directly, view it on GitHub
<https://github.com/googlegenomics/dockerflow/pull/6#issuecomment-249595842>,
or mute the thread

**Comment by jbingham** [comment]
I'm wondering if we should change this to a FAQ or Troubleshooting section, where quotas is one topic. Wdyt?  Eg, 

FAQ

What if I want to run a large batch of parallel jobs?

I got an authentication error. What should I do?

**Comment by seandavi** [comment]
Agreed.  I'd call it FAQ and Troubleshooting, just to make it clear that "both" topics are covered.

File: README.md [hunk 4]
```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
     (https://cloud.google.com/genomics/v1alpha2/pipelines).
 *   Read about [Dataflow](https://cloud.google.com/dataflow).
 *   Write your own workflows!
+
+## A word about quotas
```
Comment (jbingham) [inline README.md]: FAQ and troubleshooting

### What if I want to run a large batch of parallel jobs?

Google Cloud Platform has various quotas ...

**Comment by jbingham** [comment]
Ping. Want to make one more round and I can accept the PR?

**Pull request merged**